examples/fungible-token/tests/utils.rs

```diff
@@ -0,0 +1,195 @@
+
```

Comment:
Can common utils be put in `near-sdk` to be reusuable? What's the plan?
cc @fckt

Reply:
That's a good question. As you can see these utils here are not reusable.

At some point, we should have a general tool which generate an external client for contracts which will create transactions (or just actions) based on the contract metadata.